translations/scantailor_de.ts

```diff
@@ -397,7 +397,7 @@ zur Auswahl mehrerer Seiten.</translation>
     <message>
         <location filename="../interaction/InteractiveXSpline.cpp" line="47"/>
         <source>Click to create a new control point.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Neuen Kontrollpunkt durch klicken erstellen.</translation>
```

Comment:
... durch Klicken ...

Reply:
"klicken" is a verb, see e.g. https://www.duden.de/rechtschreibung/anklicken

Reply:
Klar ist es ein Verb. Und Verben schreibt man groß nach "durch": https://de.wiktionary.org/wiki/Verzeichnis:Deutsch/Rechtschreibung/Gro%C3%9F-_und_Kleinschreibung_von_Verben.

Reply:
Also a word which should be written in upper case: German (in the commit message).

Reply:
DPI ist übrigens auch falsch (auch im englischen Original), richtigerweise wäre es Pixel und keine Dots, d.h. PPI (pixel per inch).
Ich plädiere aber dafür, diese sprachliche Ungenauigkeit zu belassen bzw. dies auch grammatikalisch inkorrekt zu übersetzen. "DPI" verstehen die Anwender einfacher als "Pixelauflösungswert" oder andere krude Übersetzungen.
Eventuell "DPI-Wert" anstatt dem reinen "DPI" ...